src/php/utils/docker/php-fpm-healthcheck

```diff
@@ -77,7 +77,7 @@ get_fpm_status() {
 check_fpm_health_by() {
     OPTION=$(echo "$1" | sed 's/--//g; s/-/ /g;')
     VALUE_EXPECTED="$2";
-    VALUE_ACTUAL=$(echo "$FPM_STATUS" | grep "^$OPTION" | cut -d: -f2 | sed 's/ //g')
+    VALUE_ACTUAL=$(echo "$FPM_STATUS" | grep "^$OPTION:" | cut -d: -f2 | sed 's/ //g')
```

Comment:
this change is also as consequence of what's described in [this PR](https://github.com/renatomefi/php-fpm-healthcheck/pull/25) right? so the option doesn't need to be given with the `:` suffixced to it

Reply:
There's no consequence, basically this is what extracts the metric from php-fpm.
Before the regex would match `listen queue len:` and `listen queue:`, by adding the `:` to the regex it'll match the correct metric. The effect before is that wouldn't matter whether you'd be using one metric or the other, it'd always match the first one

Reply:
aaaaaaaaah, because `listen queue` would also match `listen queue len` metrics. got it 👍